### PR TITLE
[3562] removed line with shownotification event

### PR DESCRIPTION
**Pull request by pablobentivengo**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3562

**Problem:**
* Double notification when submitting email on ForgotPassword

**In this PR:**
* Deleted Line which triggered the show notification event

### CODE REVIEW

**Comment by Elena-Evsa** [comment]
Issue 1 from #3925 - failed

**Comment by Elena-Evsa** [comment]
Issue 1: fixed
Issue 2: if user tp on account sign on mobile then shown elements from forgot password page with broken layout and wrong context, but should be 'Sign in' page shown

**Comment by Elena-Evsa** [comment]
Issue 1: fixed
Issue 2: still can reproduce: forgot password page with wrong layout
Issue 3: still can reproduce: redirect to sign in absent on mobile

**Comment by Elena-Evsa** [comment]
Issue 1: fixed
Issue 2: fixed
Issue 3: fixed

File: packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.container.js [hunk 1]
```diff
@@ -141,8 +141,12 @@ export class MyAccountOverlayContainer extends PureComponent {
 
         if (isPasswordForgotSend !== currentIsPasswordForgotSend) {
             stateToBeUpdated.isPasswordForgotSend = isPasswordForgotSend;
-            // eslint-disable-next-line max-len
-            showNotification('success', __('If there is an account associated with the provided address you will receive an email with a link to reset your password.'));
+
+            if (!isOverlayVisible) {
+                showNotification('success',
```
Comment (riha112) [inline packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.container.js:146]: if you decide to split function call into multiple lines please use pattern:
```js
functionName(
    parameterOne,
    parameterTwo,
    // Comment
    parameterThree
);

// OR

// Comment
functionName(parameterOne, parameterTwo, parameterThree);
````

**Review by riha112 — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.container.js (hunk 1 above)
Comment (pablobentivengo) [reply packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.container.js:146]: Done

**Review by riha112 — APPROVED** [review]

**Pull request merged**